encrypted-media-respec.html

```diff
@@ -2216,9 +2228,21 @@
 
         <section id="encrypted-block-encountered">
           <h4>Encrypted Block Encountered</h4>
-          <p>The following steps are run when the media element encounters a block of encrypted <a def-id="media-data"></a> during the <a def-id="resource-fetch-algorithm"></a>:</p>
-
+          <p>This algorithm queues a block of encrypted media data, <var>block</var> for decryption and attempts to decrypt if possible. The following steps are run:</p>
```

Comment:
nit: comma after `<var>block</var>`.
Or define the variable in a step: `Let <var>block</var> be the block of encrypted media data.`